lib/gettext/macros.ex

```diff
@@ -709,15 +709,29 @@ defmodule Gettext.Macros do
       """
     end
 
-    # We support nil too in order to fall back to a nil context and always use the *p
-    # variants of the Gettext macros.
+    do_expand_to_binary(term, env, raiser)
```

Comment:
We kind of moved away from the convention of `do_` prefixes because of their "imperative" feel. Instead, here what if we call this
```suggestion
    validated_expand_to_binary(term, env, raiser)
```

?

Reply:
OK ! Changed 👌